src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Container,
+  Button,
+  Layout,
+  useToggle,
+} from '@edx/paragon';
+import {
+  CheckCircle, Info, WarningFilled,
+} from '@edx/paragon/icons';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { fetchCourseAppSettings, updateCourseAppSetting, fetchProctoringExamErrors } from './data/thunks';
+import { getCourseAppSettings, getSavingStatus, getProctoringExamErrors } from './data/selectors';
+import SettingCard from './setting-card/SettingCard';
+import SettingAlert from './setting-alert/SettingAlert';
+import SettingsSidebar from './settings-sidebar/SettingsSidebar';
+import { RequestStatus } from '../data/constants';
+import { parseArrayOrObjectValues } from '../utils';
+import messages from './messages';
+
+const AdvancedSettings = ({ intl, courseId }) => {
+  const advancedSettingsData = useSelector(getCourseAppSettings);
+  const savingStatus = useSelector(getSavingStatus);
+  const proctoringExamErrors = useSelector(getProctoringExamErrors);
+  const dispatch = useDispatch();
+  const [showWarningAlert, setShowWarningAlert] = useState(false);
+  // eslint-disable-next-line no-unused-vars
+  const [isOn, setOn, setOff, toggle] = useToggle(false);
+  const [settingValues, setSettingValues] = useState({});
+
+  useEffect(() => {
+    dispatch(fetchCourseAppSettings(courseId));
+    dispatch(fetchProctoringExamErrors(courseId));
+  }, [courseId]);
+
+  const handleSettingChange = (e, settingName) => {
+    const { value } = e.target;
+    if (!value) {
+      return setSettingValues((prevInputValues) => ({
+        ...prevInputValues,
+        [settingName]: ' ',
+      }));
+    }
+    if (!showWarningAlert) {
+      setShowWarningAlert(true);
+    }
+    return setSettingValues((prevSettingValues) => ({
+      ...prevSettingValues,
+      [settingName]: value,
+    }));
+  };
+
+  const handleResetSettingsValues = () => {
+    setSettingValues({});
+    setShowWarningAlert(false);
+  };
+
+  const handleUpdateAdvancedSettingsData = () => {
+    setShowWarningAlert(false);
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+    dispatch(updateCourseAppSetting(courseId, parseArrayOrObjectValues(settingValues)));
+  };
+
+  return (
+    <>
+      <Container size="xl">
+        <div className="setting-header mt-5">
+          {(savingStatus === RequestStatus.FAILED) && (
```

Comment:
This Alert should be shown only if we have errors data in `proctoringExamErrors` variable, not for every failed request